docs/data_formats.md

```diff
@@ -36,6 +36,7 @@ Message definition (JSON, named $ID)
 ```
 {
   "MessageID"           : String,           // Required, Globally unique
+  "SequenceNumber"      : Int               // Required, Unique, but not necessarily consecutive
```

Comment:
So if we do this, we're going to have to build a schema upgrade tool - as the data in Firestore production doesn't have this at the moment. I think we should probably go ahead and do that

Reply:
Yeah, sounds good, I'll open an issue for the schema upgrade tool.